src/sys/linux/random.cr

```diff
@@ -30,6 +30,21 @@ module Sys
       end
     end
 
+    # Returns a random UInt32.
+    def self.random_number : UInt32
```

Comment:
I stupidly missed the `next_u : UInt8` in your PR, and couldn't understand the granularity in `next_u` you were talking about. I renamed `random_number` to `next_u` for symmetry with `Random#next_u` and implementations now return an UInt8, except OpenBSD which calls `arc4random` and thus returns an UInt32.